assembl/views/__init__.py

```diff
@@ -362,9 +363,78 @@ def process_export_list(ls):
         providers=providers,
         providers_json=json.dumps(providers),
         translations=io.open(jedfilename, encoding='utf-8').read(),
-        admin_email=admin_email
+        admin_email=admin_email,
     )
 
+    base.update({
+        "opengraph_locale": get_opengraph_locale(request),
+        "get_description": get_description(request),
+        "get_landing_page_image": get_landing_page_image(),
+        "private_social_sharing": private_social_sharing(),
+        "get_topic": get_topic(request)
+    })
+
+    return base
+
+
+def private_social_sharing():
+
+    from ..auth.util import get_current_discussion
+    discussion = get_current_discussion()
```

Comment:
You could have avoided the use of this method and fetched the discussion from the request. However, this is fair enough.